docs/user/metacat/source/intro.rst

```diff
@@ -58,43 +58,40 @@ please see http://knb.ecoinformatics.org.
-applications required to run the software on Linux platform.
-Chapter Five covers how to configure Metacat, both for new and upgraded
-installations. Chapter Seven details how you can access and submit information easily using
-Metacatui, the default web interface that is shipped with Metacat. Chapter Eight describes
-the Metacat indexer with solr. Chapter Nine looks at Metacatui themes, and Chapter Ten discusses
-how to set up authentication as a Metacat operator. Chapter Thirteen discusses logging, and
-Chapter Fourteen contains instructions for creating a site map, which makes individual metadata
-entries available via Web searches. Additionally, Metacat's `Java API`_ is available for developers.
+This Administrator's guide provides comprehensive steps for downloading, installing, and configuring
+Metacat for Ubuntu and other Linux variants, alongside instructions for using Metacatui, setting up
+themes, authentication, logging, and creating a site map for improved web search visibility.
+Metacat can run on MacOS but it requires users' customized installation.
+
+For detailed information, refer to the table of contents. Additionally, Metacat's `Java API`_ is
+available for developers.
 
 .. _Java API: ./api/index.html
 
 Metacat Features
 ----------------
-Metacat is a repository for data and metadata (documentation about data), that 
-helps scientists find, understand and effectively use the data sets they manage or 
+Metacat is a repository for data and metadata (documentation about data), that
+helps scientists find, understand and effectively use the data sets they manage or
 that have been created by others. Specifically,
 
 * Metacat is an open source web application, which can run on Linux and MacOS operating systems and is written in Java
-* Metacat's Web interface facilitates the input and retrieval of data 
+* Metacat's Web interface facilitates the input and retrieval of data
 * Metacat's optional mapping functionality enables you to query and visualize the geographic coverage of stored data sets
 * Metacat's replication feature ensures that all Metacat data and metadata is stored safely on multiple Metacat servers
-* Metacat's default web interface can be easily modified through metacatui, which is shipped with Metacat.
+* Metacat's default web interface `metacatui`_ can be easily modified.
 * Metacat can be customized to use Life Sciences Identifiers (LSIDs), uniquely identifying every data record
 * Metacat has a built-in logging system for tracking events such as document insertions, updates, deletes, and reads
-* The appearance of Metacat's Web interface can be customized via themes. 
+* The appearance of Metacat's Web interface can be customized via themes.
 * Metacat fully supports the DataONE Member Node interface, allowing Metacat deployments to easily participate in the DataONE federation
 
+.. _metacatui: https://nceas.github.io/metacatui/docs/
```

Comment:
this might be a more-friendly landing page to use:  https://nceas.github.io/metacatui/